app/assets/stylesheets/src/shared/_scaffolds.scss

```diff
@@ -199,6 +199,7 @@ picture {
 
 .icon {
   stroke-width: 1.25;
+  vertical-align: middle;
```

Comment:
It's not exactly vertical in the center, but the default was `bottom` (for some reason) and it's definitely more centered

We can also set a numeric value and -2px for 15px icons is perfect in the center, sort of, but for larger icons a different number is needed and it looks like a HACK

Reply:
With middle:
![image](https://github.com/user-attachments/assets/4ffc5c05-8efb-476e-8930-d12a45c73c9f)

-2px:
![image](https://github.com/user-attachments/assets/b3fcb109-b434-4f6f-af70-15c827aa2733)

Reply:
maybe `sub` ?

https://github.com/user-attachments/assets/5ff7797f-24cc-4a4b-8510-4e2b997f5d28

Reply:
or even rethink compose logic to used grid system for this column

Reply:
seems `sub` is not the solution either

Reply:
Yes, I originally tried via flex, but then with a long position name it's not clear where to position this block

Reply:
i'm afraid Amgalan won't approve this solution

Reply:
using pixel hack is also bad idea.

Reply:
With sub in firefox it looks the same as with hack
![image](https://github.com/user-attachments/assets/37db6424-031f-4e02-a374-93a6e5d20aa5)

But in chromium sub and middle are the same

Reply:
Anyway, it's better than it was

Reply:
for me `sub` looks not ok for the last item
![image](https://github.com/user-attachments/assets/cee47da5-362f-445a-be17-b3f9361632a9)

Reply:
but let's try this solution